pkg/operation/botanist/component/etcd/etcd.go

```diff
@@ -185,14 +185,19 @@ func (e *etcd) Deploy(ctx context.Context) error {
 		replicas = e.computeReplicas(foundEtcd, existingEtcd)
 
 		protocolTCP             = corev1.ProtocolTCP
-		intStrPortEtcdClient    = intstr.FromInt(PortEtcdClient)
-		intStrPortBackupRestore = intstr.FromInt(PortBackupRestore)
+		intStrPortEtcdClient    = intstr.FromInt(int(PortEtcdClient))
+		intStrPortBackupRestore = intstr.FromInt(int(PortBackupRestore))
 
 		resourcesEtcd, resourcesBackupRestore = e.computeContainerResources(foundSts, existingSts)
 		quota                                 = resource.MustParse("8Gi")
 		storageCapacity                       = resource.MustParse(e.storageCapacity)
 		garbageCollectionPolicy               = druidv1alpha1.GarbageCollectionPolicy(druidv1alpha1.GarbageCollectionPolicyExponential)
 		garbageCollectionPeriod               = metav1.Duration{Duration: 12 * time.Hour}
+		compressionPolicy                     = druidv1alpha1.GzipCompression
```

Comment:
```suggestion
		compressionPolicy                     = druidv1alpha1.DefaultCompression
```
Can you add the above changes?
Although it will also do the same thing but in the future if we change the `DefaultCompression` policy, then we don't have to explicitly change the `compressionPolicy` here.

Reply:
@ishan16696 Do we want to tie the Gardener compression policy to druid's default compression policy? I think it's better we keep these separate, since we might want to have a compression policy in gardener different from druid's default policy, although I'm certain we will not change the default compression policy in druid.

Reply:
>Do we want to tie the Gardener compression policy to druid's default compression policy?

No, Actually `DefaultCompression` was declared especially for gardener usage. But either way is fine for me.